Libraries/CustomComponents/NavigationExperimental/NavigationHeader.js

```diff
@@ -186,6 +188,7 @@ class NavigationHeader extends React.Component<DefaultProps, Props, any> {
       return null;
     }
 
+    const {subviewStyle = {} } = this.props;
```

Comment:
I rather keep the definition of style simple without introducing a ternary expression and I don't think the cost of allocating an empty literal will matter at all...